test/ui-tests/title.test.js

```diff
@@ -0,0 +1,14 @@
+const timeout = process.env.SLOWMO ? 60000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+  path = fs.realpathSync('file://../index.html');
+  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+
+describe('Title of the page', () => {
+  test('Title of the page', async () => {
+    const title = await page.title();
+    expect(title).toBe('woofmark');
```

Comment:
What would be some next steps in terms of basic tests we could add? Question for everybody! cc @NitinBhasneria @shreyaa-sharmaa @keshav234156 🎉

Reply:
I will be adding the test for #2 and #3